crates/macro/src/lib.rs

```diff
@@ -5,6 +5,7 @@ extern crate proc_macro;
 use proc_macro::TokenStream;
 use quote::quote;
 
+/// A list of all the attributes can be found here: https://rustwasm.github.io/wasm-bindgen/reference/attributes/index.html
```

Comment:
This links to the docs for the master branch, we instead want to link to the stable branch (latest release).
```suggestion
/// A list of all the attributes can be found here: https://rustwasm.github.io/docs/wasm-bindgen/reference/attributes/index.html
```